doc/developers/advanced_installation.rst

```diff
@@ -0,0 +1,427 @@
+folder therefore not requiring root permission. This flag should make pip
+ignore any old version of scikit-learn previously installed on the system while
+benefiting from system packages for numpy and scipy. Those dependencies can
+be long and complex to build correctly from source.
+
+The ``--install-option="--prefix="`` flag is only required if Python has a
+``distutils.cfg`` configuration with a predefined ``prefix=`` entry.
+
+
+From source package
+~~~~~~~~~~~~~~~~~~~
+
+Download the source package from http://pypi.python.org/pypi/scikit-learn/
+, unpack the sources and cd into the source directory.
+
+This packages uses distutils, which is the default way of installing
+python modules. The install command is::
+
+    python setup.py install
+
+or alternatively (also from within the scikit-learn source folder)::
+
+    pip install .
+
+.. warning::
+
+   Packages installed with the ``python setup.py install`` command cannot
+   be uninstalled nor upgraded by ``pip`` later. To properly uninstall
+   scikit-learn in that case it is necessary to delete the ``sklearn`` folder
+   from your Python ``site-packages`` directory.
+
+
+.. _install_by_distribution:
+
+Third party distributions of scikit-learn
+=========================================
+
+Some third-party distributions are now providing versions of
+scikit-learn integrated with their package-management systems.
+
+These can make installation and upgrading much easier for users since
+the integration includes the ability to automatically install
+dependencies (numpy, scipy) that scikit-learn requires.
+
+The following is an incomplete list of Python and OS distributions
+that provide their own version of scikit-learn.
+
+
+MacPorts for Mac OSX
+--------------------
+
+The MacPorts package is named ``py<XY>-scikits-learn``,
+where ``XY`` denotes the Python version.
+It can be installed by typing the following
+command::
+
+    sudo port install py26-scikit-learn
+
+or::
+
+    sudo port install py27-scikit-learn
+
+
+Arch Linux
+----------
+
+Arch Linux's package is provided through the `official repositories
+<https://www.archlinux.org/packages/?q=scikit-learn>`_ as
+``python-scikit-learn`` for Python 3 and ``python2-scikit-learn`` for Python 2.
+It can be installed by typing the following command:
+
+.. code-block:: none
+
+     # pacman -S python-scikit-learn
+
+or:
+
+.. code-block:: none
+
+     # pacman -S python2-scikit-learn
+
+depending on the version of Python you use.
+
+
+NetBSD
+------
+
+scikit-learn is available via `pkgsrc-wip <http://pkgsrc-wip.sourceforge.net/>`_:
+
+    http://pkgsrc.se/wip/py-scikit_learn
+
+Fedora
+------
+
+The Fedora package is called ``python-scikit-learn`` for the Python 2 version
+and ``python3-scikit-learn`` for the Python 3 version. Both versions can
+be installed using ``yum``::
+
+    $ sudo yum install python-scikit-learn
+
+or::
+
+    $ sudo yum install python3-scikit-learn
+
+
+Building on windows
+===================
```

Comment:
Yeah the level of the header is off. (again, this is copy & paste, but I'll fix it).